internal/postprocessor/main.go

```diff
@@ -528,13 +544,34 @@ func (p *postProcessor) processCommit(title, body string) (string, string, error
 					scope = scopes[0]
 				}
 
-				newCommitTitle := updateCommitTitle(currTitle, scope)
+				newCommitTitle := updateCommit(currTitle, scope)
```

Comment:
Have you done an audit of how the term "title" is used here? Any thoughts about how to better align the usage with the way the term is used in git and GitHub?

Reply:
We resolved offline: it makes sense to keep using `Title` for some variables b.c it is a recognized section of the git spec (content up to the first line break in a commit's message).